app/jobs/hyrax/batch_ingest/batch_item_processing_job.rb

```diff
@@ -33,7 +41,7 @@ def config(batch_item)
 
         def notify_failed(batch_item, exception)
           batch_item.update(status: 'failed', error: exception.message)
-          # TODO: Send email
+          BatchCompleteMailer.with(batch: batch_item.batch).batch_completed_with_errors.deliver_later
```

Comment:
I think this line should be removed since we want to let the `after_perform` handle the batch complete logic and this is for updating the batch item status.  Sorry the method name and TODO are misleading.